addons/account/models/account_account.py

```diff
@@ -859,13 +862,12 @@ def _merge_method(self, destination, source):
 class AccountGroup(models.Model):
     _name = "account.group"
     _description = 'Account Group'
-    _parent_store = True
```

Comment:
Having `_parent_store = True` comes with its own mechanism to guarantee the absence of cycles.  Removing the `_parent_store` implies to re-add an explicit constraint for that.

Reply:
Oh damned, the constraint was actually present.  It was redundant, it now is useful.